dev/com.ibm.ws.app.manager.wab.installer_fat/build.gradle

```diff
@@ -10,7 +10,7 @@
  *******************************************************************************/
  
 dependencies {
-  requiredLibs 'org.apache.commons:commons-compress:1.20'
+  requiredLibs 'org.apache.commons:commons-compress:1.21'
```

Comment:
given how many places that commons-compress is used, we should probably create a test bundle for it and depend on that... `io.openliberty.org.apache.commons.compress`